decision-records/007-javascript-file-formats.md

```diff
@@ -0,0 +1,84 @@
+# JavaScript file names and formats
+
+## Decision
+
+We will consolidate all published JavaScript in the [`govuk-frontend` npm package](https://www.npmjs.com/package/govuk-frontend) into a single `govuk` directory.
+
+We will publish JavaScript in the following formats:
+
+1. ECMAScript (ES) modules
+2. ECMAScript (ES) modules, bundled
+3. Universal Module Definition (UMD), bundled
+
+We will use ECMAScript (ES) modules by default and only bundle JavaScript files that are considered "entry points" into GOV.UK Frontend, such as `all.mjs` and exported components (for example `accordion.mjs`).
+
+We will deprecate the Universal Module Definition (UMD) format in a future release.
+
+For consistency, we will clearly update our file extensions to:
+
+1. Add prefix `.bundle` to bundled JavaScript file extensions
+2. Add prefix `.min` to minified JavaScript file extensions
+
+We will add our [GitHub release](https://github.com/alphagov/govuk-frontend/releases) JavaScript to the [`govuk-frontend` npm package](https://www.npmjs.com/package/govuk-frontend).
+
+**Date:** 2023-06-27
+
+**Status:** accepted
```

Comment:
I've raised #29 to fix this in the template, but can be bump the date and status to the top of the doc, and sentence case the status?

Reply:
Spotted it, pushed 👍